libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1508,6 +1508,39 @@ bool ExpressionCompiler::visit(MemberAccess const& _memberAccess)
 			);
 			m_context << Instruction::BALANCE;
 		}
+		else if (member == "code")
+		{
+			// Stack: <address>
+			utils().convertType(
+				*_memberAccess.expression().annotation().type,
+				*TypeProvider::address(),
+				true
+			);
+
+			m_context << Instruction::DUP1 << Instruction::EXTCODESIZE;
+			// Stack post: <address> <size>
+
+			m_context << Instruction::DUP1;
+			// Account for the size field of `bytes memory`
+			m_context << u256(32) << Instruction::ADD;
```

Comment:
In some places we actually round up `bytes memory` allocation to 256-bit wordsize.

Reply:
We also should do it here.

Reply:
I think some IR features don't do that and disagree with the old codegen -- is that a bug in the IR then?

Reply:
I'm fine with either. I think it's not a strict requirement.